app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,73 @@
+class OrdersController < ApplicationController
+ # rescue_from ActiveRecord::RecordNotFound, with: :catch_not_found
+  before_action :set_order, only: %i[ show edit update destroy ]
+  
+  def index
+    @orders = Order.all
+  end
+
+  def show
+    @order = Order.find(params[:id])
+    
+  end
+
+  def new
+    @order=Order.new
+    
+   
+  end
+
+  def edit
+   @order=Order.find(params[:id])
+   #render :edit
+    end
+  
+
+  def create
+    #@order = Order.new(order_params)
+    #@order.save
+    #flash.notice = "The order record was created successfully."
+    #redirect_to @order 
+ @order = Order.new(order_params)
+
+    respond_to do |format|
+      if @order.save
+        format.html { redirect_to order_url(@order), notice: "Order was successfully created." }
```

Comment:
As you have in the `update` method, passing in the Order object directly also works and is less verbose.

```suggestion
        format.html { redirect_to @order, notice: "Order was successfully created." }
```